woocommerce/woocommerce-android
# Improve AnalyticsViewModel visitors unit tests

Summary
==========
Adds some missing unit test coverage for the Visitors and Views implementation previously introduced.

How to Test
==========
1. Simply review the introduced unit tests, make sure they're not flaky, and if there are any missing scenarios we should also cover.

Update release notes:
